include/SFML/Graphics/Transform.hpp

```diff
@@ -350,7 +349,7 @@ constexpr Vector2f operator*(const Transform& left, const Vector2f& right);
 } // namespace sf
 
 
-#endif // SFML_TRANSFORM_HPP
+
 
```

Comment:
This is an abnormal amount of spaces for the end of the file. It doesn't hurt anything and I don't think it's a huge issue, plenty of these headers had weird spacing, but this one is like 5 where the typical is 2 or 3.